### DISCUSSION TITLE
Unsupported streaming

### DISCUSSION THREAD

**Discussion by AP2020C** [aclap-dev/video-downloadhelper#1214] [Help and Q&A]
### Your question

If a platform is unsupported or malfunctioning, is there a form to ask for support to be added?

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: it
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "used_history_button": true,
    "download_directory": "C:\\Users\\me\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\me\\dwhelper"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 74,
    "never_show_successfull_dl_message": true
}

**Comment by Mattmjs** [comment]
This is the place, you would need to provide a link if it is publicly viewable. Even if it isn't and only accessible with account and
password there ways around that. Do you have detection and it fails or do you have no detection ?

**Reply by AP2020C** [reply]
Pluto.tv
Videos are intercepted, but too short to be the full content.

**Reply by Mattmjs** [reply]
The site is not available for my location so can't test it out.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

The site has DRM protection in place.

![#01](https://github.com/user-attachments/assets/f81db57f-a9e5-44b3-90e3-368bbfec5e3b)

VDH has never, does not now, nor will ever download content protected by DRM.  It would be illegal.  DRM is meant to protect copyrighted material against copying.  No legal software can download content protected by DRM.

Give up.

**Reply by AP2020C** [reply]
I understand
